docs/AboutUs.md

```diff
@@ -7,15 +7,15 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project team
 
-### John Doe
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Gordon25.png" width="200px">
```

Comment:
> The filename of the profile photo should be docs/images/githbub_username_in_lower_case.png
Note the need for lower case (why lowercase?) e.g. JohnDoe123 -> docs/images/johndoe123.png not docs/images/JohnDoe123.png.

Please change to lowercase.